did/base.py

```diff
@@ -125,6 +124,17 @@ def quarter(self):
                 f"Invalid quarter start '{month}', should be integer.")
         return month
 
+    @property
+    def week(self):
+        """ The first day of the week, 0 (Monday) by default"""
+        week = self.parser.get("general", "week", fallback=0)
+        try:
+            week = int(week) % 7
```

Comment:
I would suggest to have a string in the config rather than an in. Transferring to an int internally is fine though. I think of something like this:

```python
#!/bin/env python

WEEKDAYS = ["Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday", "Sunday"]
config_first_day_of_week = "WednES"

# Default first day of the week:
first_day_of_week = 0

# This should allow anything from: "Mon", "mon", "mond", "monda", "monday", "MON",
# but not "mo", "MO" or "M" because it is too short.
if len(config_first_day_of_week) > 3:
    for idx, weekday in enumerate(WEEKDAYS):
        if weekday.lower().startswith(config_first_day_of_week.lower()):
            first_day_of_week = idx

print(first_day_of_week)
```